MATLAB/dynamics_update.m

```diff
@@ -98,10 +98,12 @@
         g_eci=rotateframe(quat_eci_ecef,g_ecef')';
         %TODO add thruster firings forces and torques
         %TODO add drag and solar pressure forces
-        dydt(4:6)= g_eci;
+        F_envdrag = env_atmospheric_drag(t,y(1:3),y(4:6)); %drag force; input mission time in sec, position ECI m, and velocity ECI m/s
+        dydt(4:6)= g_eci + F_envdrag/(const.MASS+fuel_mass(t));
```

Comment:
I would say we should probably calculate `F_envdrag` just once outside of the function passed to our integrator. The drag force will most likely be constant over the course of our integration intervals and that would help us with performance.

This was done with the magnetic field at the location of the spacecraft on line 76.

@nhz2 @stewartaslan What do you think about this?